cluster/images/hyperkube/master.json

```diff
@@ -12,8 +12,18 @@
               "/hyperkube",
               "controller-manager",
               "--master=127.0.0.1:8080",
+              "--terminated-pod-gc-threshold=100",
```

Comment:
This is pretty low, to the point that Jobs won't run well since they use terminated pods to checkpoint their status. What is the motivation?